justfile

```diff
@@ -1,24 +1,26 @@
 just := just_executable()
+rootdir := ''
+prefix := '/usr/local'
```

Comment:
I don't think this is necessary for overriding, you just make what used to be arguments to specific recipes into global arguments.